### ISSUE TITLE
Dirvish broken after MacOS Ventura upgrade [Bug] 

### ISSUE THREAD

**Issue by tdstoff** [alexluigit/dirvish#189]
### Thank you for the bug report

- [X] I am using the latest version of `dirvish` related packages.
- [X] I checked [CUSTOMIZING|EXTENSIONS](https://github.com/alexluigit/dirvish#resources).
- [ ] You may also try reproduce the issue using clean environment and minimal configurations with
the command `emacs -Q`.

### Bug description

Dirvish is almost unusable after I did a software update to MacOS 13.1. `vc-state` no longer shows any preview of commits in the dirvish buffer and performing simple commands like `dirvish-yank` fail.

A simple configuration of the package with just `(use-package dirvish)` and `(dirvish-override-dired-mode)` prints the message "error in process sentinel: End of file during parsing" to the messages buffer any time a new file is opened with dirvish. 

Marking a file and executing `dirvish-yank` gives the message "apply: Doing vfork: No such file or directory" and fails to copy the marked file.

### Steps to reproduce

Presumably upgrade to MacOS 13.1, although I'm not able to find anyone else reporting similar errors. 

### Expected behavior

`vc-state` should give `git` information if it's activated in `dirvish-attributes` (it is) and I should be able to copy/move files with `y y`, `y m`, etc.

### OS

MacOS

### Emacs Version

29

### Emacs Configurations

Doom Emacs

### Anything else

Unfortunately neither of these errors are catchable with `toggle-debug-on-error`. I tried checking out a few older commits of dirvish and the issue remains. The "End of file during parsing" would imply some place that parens are not closed, but I'm not sure what call is leading to a function with that issue. All of the dirvish files are fine.

**Comment by kmontag** [comment]
FWIW, I'm seeing these same errors (`error in process sentinel: End of file during parsing` when creating Dirvish buffers, and `Doing vfork: No such file or directory` when doing `dirvish-yank`) on Emacs 29 on MacOS 13.0.1.

I'm able to catch the `dirvish-yank` error using `toggle-debug-on-error`. The top of the stack trace is: 

```
Debugger entered--Lisp error: (file-missing "Doing vfork" "No such file or directory")
  dirvish-yank--execute("(progn (require 'dired-aux) (require 'dired-x) (se..." (#<buffer straight> ("/Users/...") "/Users/..." dired-copy-file) batch)
  dirvish-yank-default-handler(dired-copy-file ("/Users/...") "/Users/...")
  dirvish-yank--apply(dired-copy-file nil)
  dirvish-yank(nil)
  funcall-interactively(dirvish-yank nil)
  command-execute(dirvish-yank record)
```

**Comment by isamert** [comment]
Can you check if #192 fixes your problem?

**Comment by tdstoff** [comment]
> Can you check if #192 fixes your problem?

Yes, that PR fixes all of the problems mentioned above and Dirvish once again seems to work fully. Thank you!

**Comment by anupam312nwd** [comment]
Thanks! It worked for me too. 
I just have to run `ln -Fs $(find /opt/homebrew -name "Emacs.app") /Applications/Emacs.app` since `Emacs.app` was not present/linked in my case.
